packages/design-system/src/styles/components/_Autocomplete.scss

```diff
@@ -15,6 +15,9 @@
       display: block;
     }
   }
+  input:focus {
+    @include focus-styles-position;
+  }
```

Comment:
Do we want to apply this mixin only when focus-styles is enabled? ie: check for `@if $ds-include-focus-styles` first?

Reply:
Good catch and yes I think so!

Reply:
I think this would be a good idea.